docs/en/pid-eaa-issuance.rst

```diff
@@ -914,7 +914,12 @@ Credential Response to the Wallet Instance MUST be sent using `application/json`
 Entity Configuration Credential Issuer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Below is a non-normative example of an Entity Configuration containing an `openid_credential_issuer` metadata.
+Below is a non-normative example of an Entity Configuration of a PID Provider containing a metadata for 
+
+  - `federation_entity`
+  - `oauth_authorization_server`
+  - `openid_credential_issuer`
+  - `openid_relying_party`
```

Comment:
```suggestion
  - `wallet_relying_party`
```

Reply:
For PID Provider we have openid_relying_party as the Issuer in this case uses national eID schemes to authenticate the User, right?